README.md

```diff
@@ -1,2 +1,59 @@
 # `questdb-confstr`
 
+## Format
+
+Parser for a configuration string format used by
+[QuestDB clients](https://questdb.io/docs/reference/clients/overview/).
+
+The format is as follows:
+
+```plain
+service::key1=value1;key2=value2;key3=value3;
+```
+
+A few rules:
+* The last semicolon is mandatory.
+* Service name and keys are case-insensitive.
+* Values are case-sensitive.
+* A semicolon can't appear as a key.
```

Comment:
nit: it should be _"in a key"_?

Reply:
In actual fact, I've restricted it further since then: Keys are now alphanumeric, starting with a letter.
I'll tweak it.